.github/workflows/build.yml

```diff
@@ -83,6 +83,50 @@ jobs:
           name: abi-data
           path: ./Doc/data/*.abi
 
+  check_autoconf_regen:
+    name: 'Check if Autoconf files are up to date'
+    # Don't use ubuntu-latest but a specific version to make the job
+    # reproducible: to get the same tools versions (autoconf, aclocal, ...)
+    runs-on: ubuntu-24.04
```

Comment:
Is it okay to use 24.04 at 3.14?

Reply:
It's only running docker, so it should not matter, right?

Reply:
Yes, and it'll be easier to maintain to have the same on each branch.